snowfakery/plugins.py

```diff
@@ -136,7 +136,7 @@ def __init__(self, result: Mapping):
         self.result = result
 
     def __getattr__(self, name):
-        return self.result[name]
+        return self.__dict__["result"][name]
```

Comment:
Avoids the possibility of recursive getattr if result is not defined in a subclass.